prov/efa/src/rxr/rxr_pkt_type_misc.c

```diff
@@ -471,22 +470,9 @@ int rxr_pkt_init_eor(struct rxr_ep *ep, struct rxr_rx_entry *rx_entry, struct rx
 void rxr_pkt_handle_eor_send_completion(struct rxr_ep *ep,
 					struct rxr_pkt_entry *pkt_entry)
 {
-	struct rxr_eor_hdr *eor_hdr;
 	struct rxr_rx_entry *rx_entry;
 
-	eor_hdr = (struct rxr_eor_hdr *)pkt_entry->pkt;
-
-	rx_entry = ofi_bufpool_get_ibuf(ep->rx_entry_pool, eor_hdr->rx_id);
-	assert(rx_entry && rx_entry->rx_id == eor_hdr->rx_id);
```

Comment:
Is it worth keeping this assert around?

Reply:
Yes, it is a good assertion. I will add it back.

Reply:
Added it back in new revision